src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xencoder/HaivisionXEncoderCommunicator.java

```diff
@@ -1149,14 +1225,15 @@ private void controlAudioProperty(String property, String value, Map<String, Str
 
 			extendedStatistics.put(propertyName + EncoderConstant.HASH + EncoderConstant.EDITED, EncoderConstant.TRUE);
 			extendedStatistics.put(propertyName + EncoderConstant.HASH + EncoderConstant.CANCEL, EncoderConstant.EMPTY_STRING);
-			advancedControllableProperties.add(createButton(propertyName + EncoderConstant.HASH + EncoderConstant.CANCEL, EncoderConstant.CANCEL, EncoderConstant.CANCEL, 0));
+			advancedControllableProperties.add(createButton(propertyName + EncoderConstant.HASH + EncoderConstant.CANCEL, EncoderConstant.CANCEL, EncoderConstant.CANCELING, 0));
 		}
 	}
 
 	/**
 	 * Sent request to action audio
 	 *
 	 * @param audioConfig is instance AudioConfig DTO
+	 * @throws ResourceNotReachableException if set action audio config failed
```

Comment:
Change throws ResourceNotReachableException => throws Exception because not only ResourceNotReachableException is thrown, but also CommandFailureException.